### PR TITLE
Avro serialization with service registry

### PR DESCRIPTION
**Pull request by mayorova**

### CODE REVIEW

File: src/main/java/com/jromanmartin/kafka/consumer/ConsumerListener.java [hunk 1]
```diff
@@ -17,12 +21,15 @@
     private final static Logger LOGGER = LoggerFactory.getLogger(ConsumerListener.class);
 
     @KafkaListener(
-            topics = {"one-topic", "another-topic"},
+            topics = {"sb-topic2"},
```
Comment (mayorova) [inline src/main/java/com/jromanmartin/kafka/consumer/ConsumerListener.java:24]: Just for testing...

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -1,11 +1,15 @@
+# Spring Boot server port
+server.port= 8800
```
Comment (mayorova) [inline src/main/resources/application.properties:2]: Just for testing...

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -1,11 +1,15 @@
...
 # Kubernetes or OpenShift Service
-kafka.bootstrap-servers=my-kafka-kafka-bootstrap:9092
+# kafka.bootstrap-servers=my-kafka-kafka-bootstrap:9092
+kafka.bootstrap-servers=localhost:9092
 # Local Environment
 #kafka.bootstrap-servers=rh8amq01:9092,rh8amq02:9092,rh8amq03:9092
 
 # Client and Group
-kafka.groupId	= kafkaClientSBConsumerGroup
+kafka.groupId	= kafkaClientSBConsumerGroup3
```
Comment (mayorova) [inline src/main/resources/application.properties:12]: Just for testing...

**Comment by rmarting** [comment]
@mayorova I will close this PR because I created a new [branch](https://github.com/rmarting/kafka-clients-sb-sample/tree/feature/avro-and-service-registry-integration) with a full sample with the following features:

* Using Avro to declare schemas
* Send messages using Avro Serializers integrated with a Service Registry (Apicurio)
* Consume messages using Avro Deserializers integrated with a Service Registry
* Using different strategies to declare the message schema (Simple, Topic)
* Using Avro Maven Plugin to generate classes from avsc files
* Using Apicurio Maven Plugin to register schemas in Service Registry
* Deploy Kafka cluster and Apicurio Register using Kubernetes operators

I hope you find this new branch helpful. I will work on it some days more to add/refactor something else and be more "productive" sample.

Any comment or idea is welcomed.

**Pull request closed without merging**